mmcv/ops/csrc/pytorch/nms_quadri.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+#include "pytorch_cpp_helper.hpp"
+
+Tensor nms_quadri_cpu(const Tensor dets, const Tensor scores,
```

Comment:
Could we register the implementation in `cpu/nms_quadri.cpp` and `cuda/cudabind.cpp`?

Reply:
The input parameters of cpu and cuda is different (mmcv/ops/csrc/pytorch/nms_quadri.cpp). So we cannot register it as same input.